jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableCommon.kt

```diff
@@ -65,6 +65,30 @@ class ExecutableCommon {
             }
         }
 
+        @JvmStatic
+        fun checkSemVerVersionForParallelValidVersions(
+            version: SemVer,
+            versionRange: List<ExecutableVersionRange>,
+            executableName: String
+        ) {
+            // for use of AWS CLI version 1 and version 2 in ECS Exec
+
+            val versionRanges = versionRange.joinToString(separator = " / ", transform = { "${it.min}  ≤ version < ${it.max}" })
+            val versionOutOfRangeMessage = message(
+                "executableCommon.version_range_wrong",
+                executableName,
+                versionRanges,
+                version
+            )
```

Comment:
We can move this below the loop as a small optimization since we dont need it in a happy case

Reply:
done